elyra/metadata/schemas/component-registry.json

```diff
@@ -38,11 +38,11 @@
           "title": "Category Names",
           "description": "Category names associated with this Component Registry (the components defined in this registry will be organized in the component palette according to these categories)",
           "type": "array",
-          "items": [
-            {
-              "type": "string"
-            }
-          ],
+          "uniqueItems": true,
+          "items": {
+            "type": "string",
+            "maxLength": 18
```

Comment:
Is this value based on anything definitive?  (I ask purely out of curiosity since it was made two characters shorter in a follow-on commit.)